TrafficCapture/dockerSolution/src/main/docker/migrationConsole/lib/console_link/tests/data/invalid_tuple.json

```diff
@@ -0,0 +1,49 @@
+{
```

Comment:
nit - how is this tuple invalid?  Maybe rename the file to be slightly more specific.